templates/beegfs-oss-tuning.sh.j2

```diff
@@ -5,6 +5,7 @@ set -x
 
 echo always > /sys/kernel/mm/transparent_hugepage/enabled
 echo always > /sys/kernel/mm/transparent_hugepage/defrag
+{% if beegfs_oss_tunable | length > 0 %}
```

Comment:
previously had default([])

Reply:
It ought to default to [] if there are no `beegfs_oss_tunable` as its defined in defaults/main.yml